### DISCUSSION TITLE
Customize Episode Search String

### DISCUSSION THREAD

**Discussion by dotsam** [pymedusa/Medusa#9898] [Ideas & Feature requests]
I've got a show/provider that's picky about giving results with whitespace vs periods to match results.

Specifically, this is searching for Jeopardy (in aired-by date mode) on btdig.com (set up as a Jackett provider)
https://btdig.com/search?order=2&q=Jeopardy+2021.09.20
vs
https://btdig.com/search?order=2&q=Jeopardy+2021+09+20

The generic provider unilaterally uses the airdate, converting dashes to spaces
https://github.com/pymedusa/Medusa/blob/21e350cd4a21f6fb65f750ab5c2d1e9c92c621d6/medusa/providers/generic_provider.py#L591-L594

So the feature request here is the ability to use placeholders to customize the search string used for a series in the same way that we can customize post-processing renames.

**Comment by p0psicles** [comment]
#3732
